openslides_backend/action/actions/poll/create.py

```diff
@@ -36,7 +37,6 @@ class PollCreateAction(CreateAction):
                 "type": "array",
                 "items": options_schema,
                 "minItems": 1,
-                "uniqueItems": True,
```

Comment:
Isn't this check still right? Why did you remove it?

Reply:
It is still (or again) right, but not sufficient. The new check is stricter and therefore misleading. This check active we have 2 different error-messages for the same error from user/developer view.

Reply:
Ah okay, I see. So for consistent error messages we remove it here.